command/monitor/monitor.go

```diff
@@ -0,0 +1,150 @@
+package monitor
+
+import (
+	"fmt"
+	"sync/atomic"
```

Comment:
You might want to look at Uber's atomic package (https://pkg.go.dev/go.uber.org/atomic?tab=doc) as it provides a nice interface to atomic values.

Reply:
This is the second suggestion I've received to check out that library and possibly put it to use.  I'll definitely do so :grin: